metricbeat/docs/developer-guide/metricset-details.asciidoc

```diff
@@ -208,6 +209,11 @@ To run all the tests, run `make testsuite`. To only run unit tests, run
 `make unit-tests` or for integration tests `make integration-tests-environment`. Be aware that
 a running Docker environment is needed for integration and system tests.
 
+Sometimes you may just want to run specify single integration test, such as `apache` module, firstly, start the docker service by run
```

Comment:
I'd replace the text with the following content (includes formatting and editorial changes). Note that each sentence beginning with a period (.) becomes a numbered list item.

````
Sometimes you may want to run a single integration test, for example, to test a
module such as the `apache` module. To do this, you can:

. Start the Docker service by running
`docker-compose run -p port:port apache`. You can skip this step if, like the
`golang` module, your module doesn't need a Docker service.

. Run `cd tests/system` to change to the folder that contains the integration
tests.

. Run `INTEGRATION_TESTS=true nosetests test_apache.py`,
remembering to replace `test_apache.py` with your own test file.

````